### PR TITLE
Added install-simple tests.

### PR DESCRIPTION
**Pull request by goofiva**
[root@eead1c0d1612 workspace]# kitchen verify --test-base-path="test/integration"
-----> Starting Kitchen (v1.24.0)
The state file either has no outputs defined, or all the defined
outputs are empty. Please define an output in your configuration
with the `output` keyword and run `terraform refresh` for it to
become available. If you are using interpolation, please verify
the interpolated value is not empty. You can use the
`terraform console` command to assist.
-----> Verifying <install-simple-local>...
$$$$$$ Running command `terraform workspace select kitchen-terraform-install-simple-local` in directory /workspace/test/fixtures/install_simple
$$$$$$ Running command `terraform output -json` in directory /workspace/test/fixtures/install_simple
server: Verifying host 10.150.0.59
Skipping profile: 'inspec-gcp' on unsupported platform: 'ubuntu/18.04'.

Profile: install_simple
Version: (not specified)
Target:  ssh://ubuntu@10.150.0.59:22

  ✔  server: Forseti server instance resources
     ✔  Command: `forseti` should exist
     ✔  Command: `forseti server configuration get` exit_status should eq 0
     ✔  Command: `forseti inventory list` exit_status should eq 0
     ✔  Command: `forseti_server` should exist
     ✔  Command: `forseti_enforcer` should exist
     ✔  Command: `python3 -m pip show forseti-security|grep Version` exit_status should eq 0
     ✔  Command: `python3 -m pip show forseti-security|grep Version` stdout should match "Version: 2.24.0"
     ✔  Command: `python3 -m pip show forseti-security|grep Version` stderr should cmp == ""
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml should exist
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory configures cai_api_timeout
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory configures inventory_retention_days
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures admin_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures admin_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures admin_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures appengine_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures appengine_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures appengine_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures bigquery_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures bigquery_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures bigquery_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudasset_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudasset_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudasset_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudbilling_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudbilling_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures cloudbilling_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures compute_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures compute_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures compute_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures container_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures container_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures container_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures crm_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures crm_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures crm_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures groups_settings_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures groups_settings_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures groups_settings_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures iam_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures iam_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures iam_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures logging_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures logging_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures logging_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures servicemanagement_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures servicemanagement_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures servicemanagement_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures serviceusage_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures serviceusage_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures serviceusage_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures sqladmin_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures sqladmin_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures sqladmin_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml inventory api_quota configures storage_disable_polling
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures audit_logging_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures bigquery_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures blacklist_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures bucket_acl_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures cloudsql_acl_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures enabled_apis_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures firewall_rule_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures forwarding_rule_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures group_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures iam_policy_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures iap_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures instance_network_interface_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures ke_scanner_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures ke_version_scanner_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures kms_scanner_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures lien_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures location_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures log_sink_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures resource_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml scanner configures service_account_key_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier configures cscc_violations_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier configures cscc_source_id
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier configures inventory_gcs_summary_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier configures inventory_email_summary_enabled
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier api_quota configures securitycenter_max_calls
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier api_quota configures securitycenter_period
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures iam_policy_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures audit_logging_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures blacklist_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures bigquery_acl_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures buckets_acl_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures cloudsql_acl_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures enabled_apis_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures firewall_rule_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures forwarding_rule_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures ke_version_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures ke_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures kms_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures groups_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures instance_network_interface_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures iap_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures lien_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures location_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures log_sink_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures resource_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures service_account_key_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml notifier resources configures external_project_access_violations_should_notify
     ✔  File /home/ubuntu/forseti-security/rules/audit_logging_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/audit_logging_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/bigquery_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/bigquery_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/blacklist_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/blacklist_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/bucket_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/bucket_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/cloudsql_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/cloudsql_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/enabled_apis_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/enabled_apis_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/external_project_access_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/external_project_access_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/firewall_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/firewall_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/forwarding_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/forwarding_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/group_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/group_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/iam_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/iam_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/iap_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/iap_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/instance_network_interface_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/instance_network_interface_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/ke_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/ke_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/ke_scanner_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/ke_scanner_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/lien_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/lien_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/location_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/location_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/log_sink_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/log_sink_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/resource_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/resource_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/retention_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/retention_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/role_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/role_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/service_account_key_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/service_account_key_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/groups_settings_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/groups_settings_rules.yaml is valid YAML
     ✔  File /home/ubuntu/forseti-security/rules/kms_rules.yaml should exist
     ✔  File /home/ubuntu/forseti-security/rules/kms_rules.yaml is valid YAML
     ✔  Command: `sudo gcloud sql instances describe forseti-server-db-608656cc` exit_status should eq 0
     ✔  Command: `sudo gcloud sql instances describe forseti-server-db-608656cc` stdout should match "- name: net_write_timeout\n    value: '240'"

Profile Summary: 1 successful control, 0 control failures, 0 controls skipped
Test Summary: 152 successful, 0 failures, 0 skipped
forseti-client: Verifying host 10.150.0.60
Skipping profile: 'inspec-gcp' on unsupported platform: 'ubuntu/18.04'.
.........

Profile: install_simple
Version: (not specified)
Target:  ssh://ubuntu@10.150.0.60:22

  ✔  client: Forseti client instance resources
     ✔  Command: `forseti` should exist
     ✔  Command: `forseti config show` exit_status should eq 0
     ✔  Command: `forseti config show` stdout should match /10.150.0.59:50051/
     ✔  Command: `forseti inventory list` exit_status should eq 0
     ✔  Command: `python3 -m pip show forseti-security|grep Version` exit_status should eq 0
     ✔  Command: `python3 -m pip show forseti-security|grep Version` stdout should match "Version: 2.24.0"
     ✔  Command: `python3 -m pip show forseti-security|grep Version` stderr should cmp == ""
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_client.yaml should exist
     ✔  File /home/ubuntu/forseti-security/configs/forseti_conf_client.yaml sets the hostname to the Forseti server IP

Profile Summary: 1 successful control, 0 control failures, 0 controls skipped
Test Summary: 9 successful, 0 failures, 0 skipped
       Finished verifying <install-simple-local> (0m31.93s).
-----> Kitchen is finished. (0m35.39s)

### CODE REVIEW

**Comment by gkowalski-google** [comment]
How does Kitchen know to run these tests? I thought we would need to add a kitchen test suite into the kitchen config like the simple_example: https://github.com/forseti-security/terraform-google-forseti/blob/master/.kitchen.yml#L27. The tests did not get run for this PR because there was a lint error on [this change](https://github.com/forseti-security/terraform-google-forseti/pull/408/files#diff-577c2e119cf7bfcb1dcbac16cfe00209R59). The `=` have to be aligned. Can you please fix that? We will see if this test suite gets run, but I think we need that change to .kitchen.yml.

File: test/fixtures/install_simple/versions.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+terraform {
+  required_version = ">= 0.12"
```
Comment (hshin-g) [inline test/fixtures/install_simple/versions.tf]: Should we hardcode this to just 0.12?

Comment (gkowalski-google) [reply test/fixtures/install_simple/versions.tf]: Discussed with @hshin-g, updating the Terraform versions will happen with this ticket: https://github.com/forseti-security/terraform-google-forseti/issues/364

**Comment by gkowalski-google** [comment]
Resolves #397

**Pull request merged**